charts/zookeeper-operator/templates/clusterrole.yaml

```diff
@@ -13,15 +13,28 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - nodes
```

Comment:
why is `nodes` got removed from resources?

Reply:
it's not needed by the operator - see https://github.com/pravega/zookeeper-operator/blob/master/deploy/all_ns/rbac.yaml